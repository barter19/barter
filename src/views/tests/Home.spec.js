import { shallow } from '@vue/test-utils';
import HomeComponent from '../Home';

const renderer = require('vue-server-renderer').createRenderer();

describe('Home.vue Component', () => {
  const ClonedComponent = shallow(HomeComponent);
  const { vm } = ClonedComponent;

  it('renders to a nice snapshot', () => {
    renderer.renderToString(vm, (err, str) => {
      expect(str).toMatchSnapshot();
    });
  });
});
