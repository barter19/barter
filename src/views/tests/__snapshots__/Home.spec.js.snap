// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue Component renders to a nice snapshot 1`] = `
<main>
  <!---->
</main>
`;
